axonserver/src/test/java/io/axoniq/axonserver/message/command/LoadFactorBasedSelectorTest.java

```diff
@@ -0,0 +1,96 @@
+package io.axoniq.axonserver.message.command;
+
+import io.axoniq.axonserver.message.command.hashing.ConsistentHashRoutingSelector;
+import org.junit.*;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+import java.util.concurrent.atomic.AtomicInteger;
+
+import static org.junit.Assert.*;
+
+/**
+ * Unit test for {@link io.axoniq.axonserver.message.command.hashing.ConsistentHashRoutingSelector}
+ *
+ * @author Sara Pellegrini
+ */
+public class LoadFactorBasedSelectorTest {
+
+    @Test
+    public void testDistribution() {
+        ConsistentHashRoutingSelector testSubject = new ConsistentHashRoutingSelector(Integer::parseInt);
+        testSubject.register("5");
+        testSubject.register("15");
+        testSubject.register("30");
+        testSubject.register("50");
+        Map<String, AtomicInteger> counter = new HashMap<>();
+        int messages = 1_000_000;
+        for (int i = 0; i < messages; i++) {
+            String handler = testSubject.selectHandler(UUID.randomUUID().toString()).orElse(null);
+            counter.computeIfAbsent(handler, h -> new AtomicInteger()).incrementAndGet();
+        }
+        System.out.println(counter);
+        counter.forEach((handler, count) -> assertEquals(Double.parseDouble(handler),
```

Comment:
I do not think that this assertion is overly clear on face value. Would you mind extracting the `count.get() * 100d / messages` outcome to a named variable, just so others can immediately see without required deduction what's being asserted against?

Reply:
yes, you're right. I'm going to extract a variable named `percentage`

Reply:
Thank you! :-)